### PR TITLE
Reduce initial learning rate for SGD example scripts

### PR DESCRIPTION
**Pull request by rotx-maxim**

### CODE REVIEW

**Comment by rotx-maxim** [comment]
This will be addressed when PyTorch 2 is merged. Closing this commit.
